C#2_domácí úkol č.1_1/Program.cs

```diff
@@ -8,33 +8,32 @@ static void Main(string[] args)
             // Napište program, který se zeptá na dvě čísla a zobrazí jejich součet.
 
             Console.WriteLine("Tento program vypíše součet dvou zadaných čísel.");
+            double cislo1;
+            double cislo2;
 
             // zadání prvního čísla
-            Console.Write("Zadej první číslo: ");
-            string input1 = Console.ReadLine();
-            bool prevedeniInput1 = double.TryParse(input1, out double cislo1);
-
-            while (!prevedeniInput1)
-            {
-                Console.Write("Špatně zadaný vstup! Zkus zadat první číslo znovu: ");
-                input1 = Console.ReadLine();
-                prevedeniInput1 = double.TryParse(input1, out cislo1);
-            }
+            cislo1 = ZadejCislo("první");
 
             // zadání druhého čísla
-            Console.Write("Zadej druhé číslo: ");
-            string input2 = Console.ReadLine();
-            bool prevedeniInput2 = double.TryParse(input2, out double cislo2);
+            cislo2 = ZadejCislo("druhé");
 
-            while (!prevedeniInput2)
+            // výpis součtu obou čísel
+            Console.WriteLine($"Součet čísel {cislo1} a {cislo2} je {cislo1 + cislo2}");          
+
+        }
+        static double ZadejCislo(string poradiCisla)
+        {
+            bool parsedInput;
+            double cislo;
+
+            do
             {
-                Console.Write("Špatně zadaný vstup! Zkus zadat druhé číslo znovu: ");
-                input2 = Console.ReadLine();
-                prevedeniInput2 = double.TryParse(input2, out cislo2);
-            }
+                Console.Write($"Zadej {poradiCisla} číslo: ");
+                parsedInput = double.TryParse(Console.ReadLine(), out cislo);
 
-            // výpis součtu obou čísel
-            Console.WriteLine($"Součet čísel {cislo1} a {cislo2} je {cislo1 + cislo2}");
+            } while (!parsedInput);
```

Comment:
This version is more readable. 

```suggestion
            } while (parsedInput is false);
```